dev/NavigationView/NavigationView.xaml

```diff
@@ -263,13 +263,12 @@
                                     <Button 
                                         x:Name="TopNavOverflowButton"
                                         Grid.Column="4"
-
                                         Content="More"
                                         Style="{StaticResource NavigationViewOverflowButtonStyleWhenPaneOnTop}"
                                         Visibility="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.OverflowButtonVisibility}">
 
                                         <Button.Flyout>
-                                            <Flyout Placement="Bottom">
+                                            <Flyout Placement="Bottom" ElementSoundMode="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=ElementSoundMode}">
```

Comment:
Why not TemplateBinding?

Reply:
Using TemplateBinding was not being resolved properly here. I am unsure why this was happening, but falling back to this regular binding works as expected.